runtime/src/accounts.rs

```diff
@@ -229,18 +230,35 @@ impl Accounts {
     }
 
     /// If feature `cap_transaction_accounts_data_size` is active, total accounts data a
-    /// transaction can load is limited to 64MiB to not break anyone in Mainnet-beta today.
-    /// (It will be set by compute_budget instruction in the future to more reasonable level).
+    /// transaction can load is limited to
+    ///   if `set_tx_loaded_accounts_data_size` instruction is not activated or not used, then
+    ///     default value of 64MiB to not break anyone in Mainnet-beta today
+    ///   else
+    ///     user requested loaded accounts size.
```

Comment:
had a dumb question, can the load size be bigger than 64MiB?

Reply:
no, 64MB is the max defined in compute-budget